### PR TITLE
Added library in ES5 and ES6 syntaxes

### PR DESCRIPTION
**Pull request by OlegAdamovich**

### CODE REVIEW

File: es5.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        take: function (array, num) {
+            var result = [];
+            for (var i = 0; i < num; i++) {
+                result.push(array[i])
+            }
+            return result;
+        },
+        skip: function (array, num) {
+            for (var i = 0; i < num; i++) {
+                array.shift();
```
Comment (MakaryHryshkin) [inline es5.js:24]: without shift. please

File: es5.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            } else {
+                return callback.apply(null, arguments);
+            }
+        }
+    }.bind(methods);
+
+    const noneChainableMethodsKeys = ['chain', 'value'];
+
+    for (var key in methods) {
+        if (methods.hasOwnProperty(key)) {
```
Comment (MakaryHryshkin) [inline es5.js:73]: What is the purpose of this condition?

Comment (OlegAdamovich) [reply es5.js:73]: To determine whether an object has the specified property as a direct property of that object.

File: es5.js (hunk 1 above)
Comment (OlegAdamovich) [reply es5.js:24]: Redone to for loop.

File: es5.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                callback(array[i]);
+            }
+        },
+    };
+
+    var chainable = function (callback) {
+        return function () {
+            if (isChaining) {
+                var args = Array.prototype.slice.call(arguments);
+                bufferValue = callback(bufferValue, args);
```
Comment (mhryshkin) [inline es5.js:63]: `callback(bufferValue, args)` did you test it?

Comment (mhryshkin) [inline es5.js:63]: ` callback(bufferValue, args)` did you test args?

File: es6.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    const chainable = function (callback) {
+        return (...args) => {
+            if (isChaining) {
+                bufferValue = callback(bufferValue, ...args);
+                return this;
+            } else {
+                return callback(...args);
+            }
+        };
+    }.bind(methods);
```
Comment (mhryshkin) [inline es6.js:68]: purpose of this bind?

**Comment by mhryshkin** [comment]
@OlegAdamovich Could you please add check args of every function to avoid unpredicted results, please use errors.

File: es6.js (hunk 4 above)
Comment (OlegAdamovich) [reply es6.js:68]: Otherwise `this` of `chainable`  will be global object `Window`

File: es5.js (hunk 3 above)
Comment (OlegAdamovich) [reply es5.js]: Fixed

**Review by mhryshkin — APPROVED** [review]
good work

**Pull request merged**